data/upstream.yaml

```diff
@@ -187,6 +199,10 @@ jabber-roster:
   repo: https://github.com/kparal/jabber-roster
   note: |
     Porting will start if deps are available.
+konversation:
+    status: in-progress
+    links:
+        homepage: https://konversation.kde.org/
```

Comment:
I'd like to use the in-progress status as little as possible*, since for anybody who wants to help it's practically the same as idle. I think it would be much more useful to summarize the above into `notes` and keep it idle.

* This wasn't always the case, there's a lot of garbage in in-progress currently :(